packages/lesswrong/components/posts/PostsItemTrailingButtons.tsx

```diff
@@ -89,7 +91,7 @@ const PostsItemTrailingButtons = ({
     <>
       {(showDismissButton || resumeReading || !isEAForum) && <div className={classes.actions}>
         <DismissButton {...{showDismissButton, onDismiss}} />
-        {!isEAForum && !resumeReading && <PostActionsButton post={post} vertical />}
+        {!isEAForum && !resumeReading && currentUser && <PostActionsButton post={post} vertical />}
```

Comment:
In the LW post item, I actually kept the three dots menu **hidden** from logged out users, because I was unsure if you'd want it. If you remove the `currentUser` check then this is what the menu looks like for logged out users:

<img width="576" alt="Screen Shot 2023-06-16 at 9 31 57 PM" src="https://github.com/ForumMagnum/ForumMagnum/assets/9057804/6b364060-2926-4cd9-9e5c-539d2f4488d8">

@jimrandomh I'll let you decide if you want to remove this check. Separately, you might want to remove the "Share" option from the dropdown in the post item context.